### PR TITLE
snapshot fast follows

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr updates the snapshot docs by:
- updating any typos
- adding some config fields
- ordering the config order to match the table
- rewording the callout

### CODE REVIEW

File: website/docs/docs/build/snapshots.md [hunk 1]
```diff
@@ -52,20 +52,25 @@ It is not possible to "preview data" or "compile sql" for snapshots in dbt Cloud
...
+    relation: source # source('my_source', 'my_table')
     config:
-      schema: snapshots
-      database: analytics
-      unique_key: id
-      strategy: timestamp
-      updated_at: updated_at
+      [database](/reference/resource-configs/database): string
+      [schema](/reference/resource-configs/schema): string
+      [alias](/reference/resource-configs/alias):string
```
Comment (mirnawong1) [inline website/docs/docs/build/snapshots.md]:
```suggestion
      [alias](/reference/resource-configs/alias): string
```

File: website/docs/docs/build/snapshots.md [hunk 2]
```diff
@@ -52,20 +52,25 @@ It is not possible to "preview data" or "compile sql" for snapshots in dbt Cloud
...
-      strategy: timestamp
-      updated_at: updated_at
+      [database](/reference/resource-configs/database): string
+      [schema](/reference/resource-configs/schema): string
+      [alias](/reference/resource-configs/alias): string
+      [strategy](/reference/resource-configs/strategy): timestamp | check
+      [unique_key](/reference/resource-configs/unique_key): column_name_or_expression
+      [check_cols](/reference/resource-configs/check_cols): [column_name] | all
+      [updated_at](/reference/resource-configs/updated_at): column_name
+      [invalidate_hard_deletes](/reference/resource-configs/invalidate_hard_deletes): true | false
```
Comment (mirnawong1) [inline website/docs/docs/build/snapshots.md:71]: @dbeatty10 i added the meta column name here and in the table below so users would explicitly know this is an option - do you think that's ok?

File: website/docs/docs/build/snapshots.md [hunk 3]
```diff
@@ -52,20 +52,25 @@ It is not possible to "preview data" or "compile sql" for snapshots in dbt Cloud
...
 
 <File name='snapshots/orders_snapshot.yml'>
-  
+
 ```yaml
 snapshots:
-  - name: orders_snapshot
-    relation: source('jaffle_shop', 'orders')
+  - name: string
+    relation: source # source('my_source', 'my_table')
```
Comment (dbeatty10) [inline website/docs/docs/build/snapshots.md]: What do you think about this to make it more clear that `relation` can be ref or source?

```suggestion
    relation: relation # source('my_source', 'my_table') or ref('my_model')
```

File: website/docs/docs/build/snapshots.md (hunk 2 above)
Comment (dbeatty10) [reply website/docs/docs/build/snapshots.md:71]: 🧠

**Review by dbeatty10 — APPROVED** [review]
One non-blocking suggestion

File: website/docs/docs/build/snapshots.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/build/snapshots.md]: oh great shout @dbeatty10 !

**Comment by mirnawong1** [comment]
thanks @dbeatty10 -folded in your sugg and shipping this puppy!

**Pull request merged**